# Generated by Django 2.1.4 on 2019-01-07 05:53

from django.db import migrations, models
import django.db.models.deletion


class Migration(migrations.Migration):

    dependencies = [
        ('tnadmin', '0003_custom_migr_add_gvfederation_data'),
    ]

    operations = [
        migrations.CreateModel(
            name='LdapSyncErrorPush',
            fields=[
                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('ldap_dn', models.CharField(max_length=222)),
                ('op', models.CharField(max_length=6)),
                ('message', models.CharField(max_length=5000)),
            ],
        ),
        migrations.CreateModel(
            name='LdapSyncJobPush',
            fields=[
                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('started_at', models.DateTimeField(auto_now_add=True)),
                ('add_upd_db_records_read', models.IntegerField(default=0)),
                ('add_upd_records_skipped', models.IntegerField(default=0)),
                ('add_upd_records_added', models.IntegerField(default=0)),
                ('add_upd_records_updated', models.IntegerField(default=0)),
                ('add_upd_records_update_failed', models.IntegerField(default=0)),
                ('del_ldap_records_read', models.IntegerField(default=0)),
                ('del_records_deleted', models.IntegerField(default=0)),
                ('del_records_delete_failed', models.IntegerField(default=0)),
            ],
        ),
        migrations.AddField(
            model_name='ldapsyncerrorpush',
            name='job_id',
            field=models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='tnadmin.LdapSyncJobPush'),
        ),
    ]
